docs/examples/DataSet/Measuring X as a function of time.ipynb

```diff
@@ -0,0 +1,179 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Measuring X as a function of time\n",
+    "\n",
+    "Sometimes we'd like to measure something as a function of elapsed wall clock time. This notebook provides a simple example of doing that using the `ElapsedTimeParameter`."
```

Comment:
I'd recommend to add here the reason behind `ElapsedTimeParameter` - the fact that qcodes provides it, hence all qcodes users in principle can use the same parameter which makes it easier to relate measurements which use this parameter (sorry, on friday evening my language skills are gone, but i hope you got what i mean :) )

Reply:
I have added a little bit. I'm not too sure what you are after... -do we need to document why standardization is helpful?

Reply:
yes, in a subtle way, like "this parameter provides a single way of measuring time which is awesome because standardization is awesome in this case" :)